docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,213 @@
+    
+    4. Return to the WSL interface, navigate to the root of your repository, and run your first scan:
+
+        ```console
+        semgrep ci
+
+        # Alternatively, run an offline-only SAST scans; no SCA scan is performed
+        # with this command
+        semgrep scan --config=auto
+        ```
+
+    </TabItem>
+
+    <TabItem value='Docker'>
+
+    :::info Prerequisites
+    Ensure that you have [Docker installed](https://docs.docker.com/desktop/) before proceeding.
+    :::
+
+    1. Pull the latest image and confirm the version:
+
+        ```console
+        docker pull returntocorp/semgrep
+
+        # confirm version
+        docker run --rm returntocorp/semgrep semgrep --version
+        ```
+
+    2. For users running Docker on **macOS or Linux** Docker:
+
+        1. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+            ```console
+            docker run -it returntocorp/semgrep semgrep login
+            ```
+
+        2. In the **Semgrep CLI login**, click **Activate** to proceed. Return to the CLI and copy the login token that's shown.
+
+        3. Navigate into the root of your repository, and run your first scan. Be sure to substitute <code><span class="placeholder">YOUR_TOKEN</span></code> with the login token value you copied in the previous step: 
+
+            ```console
+            docker run -e SEMGREP_APP_TOKEN=YOUR_TOKEN --rm -v "${PWD}:/src" returntocorp/semgrep semgrep ci
+
+            # Alternatively, run an offline-only SAST scans; no SCA scan is performed
+            # with this command
+            docker run -e SEMGREP_APP_TOKEN=YOUR_TOKEN --rm -v "${PWD}:/src" returntocorp/semgrep semgrep scan --config=auto
+            ```
+
+            The provided `-v` option mounts the current directory into the container to be scanned. Navigate into a different repository or provide a specific local directory in the command to scan a different project.
+    
+    3. For users running Docker on **Windows**:
+
+        1. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+            ```console
+            docker run -it returntocorp/semgrep semgrep login
+            ```
+
+        2. In the **Semgrep CLI login**, click **Activate** to proceed. Return to the CLI, and copy the login token that's shown.
+
+        3. Navigate into the root of your repository, and run your first scan. Be sure to substitute <code><span class="placeholder">YOUR_TOKEN</span></code> with the login token value you copied in the previous step: 
+
+            ```console
+            docker run -e SEMGREP_APP_TOKEN=YOUR_TOKEN --rm -v "%cd%:/src" returntocorp/semgrep semgrep ci
+            
+            # Alternatively, run an offline-only SAST scans; no SCA scan is performed
```

Comment:
```suggestion
            # Alternatively, run an offline-only SAST scan; no SCA scan is performed
```

I see that you put it here, but it sort of negates the whole flow because the user doesn't have to sign in for this. I apologize - maybe better to revert in keeping in line with product.